Yosemite/YosemiteTests/Stores/AccountStoreTests.swift

```diff
@@ -624,42 +553,6 @@ final class AccountStoreTests: XCTestCase {
         XCTAssertEqual(network.requestsForResponseData.count, 0)
     }
 
-    func test_loadAndSynchronizeSite_fetches_from_remote_if_forcedUpdate_is_true() throws {
```

Comment:
This is the other test I'm having troubles with, `XCTAssertEqual(viewStorage.countObjects(ofType: Storage.Site.self), 0)` is returning 2 objects, which are possibly the 2 sites that can be seen in `sites.json`?

Added to this, the rest of assertions carry these extra objects, so for example the last one counts 3 rather than 2:

`XCTAssertEqual(network.requestsForResponseData.count, 1)`

Reply:
hmm, from my testing on `trunk` and `isJetpackConnectionPackageSupported: true`, the failure is on the last line of the test case https://github.com/woocommerce/woocommerce-ios/pull/7653/files#diff-1a85b533272686060180636e639f6d520e702d4fe7ad9f787a8517b02d3b2a68L660 where the number of API requests is now 3 (2 extra requests for the JCP site, the second site in `sites.json`). I think we can just update this line and keep the test case?

The first `XCTAssertEqual(viewStorage.countObjects(ofType: Storage.Site.self), 0)` was to make sure there are no sites in storage. Then `XCTAssertEqual(self.viewStorage.countObjects(ofType: Storage.Site.self), 1)` was to ensure there is one site added to the storage after `accountStore.upsertStoredSitesInBackground` in the setup.

After `AccountAction.loadAndSynchronizeSite`, because `forcedUpdate` is `true`, this test case is to make sure `me/sites` and any extra API requests for JCP sites are still made and therefore the sites are updated in storage afterward.

Reply:
> the number of API requests is now 3 (2 extra requests for the JCP site, the second site in sites.json). I think we can just update this line and keep the test case?

Right, the 2 extra calls that we're making for JCP sites that appeared as well in the previous test as well. Fixed here: 1a159e811d8d256d49bfd180fb64098429dd22a1